### PR TITLE
#169817502 User should be able to search

### PR DESCRIPTION
**Pull request by erickyvand**
#### What does this PR do?
- Search the trip request
#### Description of Task to be completed?
- Requester search a particular trip he has requested
- Manager search for a particular request 
#### How should this be manually tested?
- Login to Barefoot Nomad
- On sidebar click on my request
- Enter the text in the search field
#### What are the relevant pivotal tracker stories?
[#169817502](https://www.pivotaltracker.com/story/show/169817502)
#### Screenshots
Screen Shot 2020-03-25 at 15 20 09

### CODE REVIEW

File: src/tests/components/trips/search.test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-undef */
+import searchReducer from '../../../redux/reducers/searchReducer';
+
+describe('Comment Component', () => {
```
Comment (bdushimi) [inline src/tests/components/trips/search.test.js]: @erickyvand is the comment component responsible to display the search results?

File: src/views/approvalsView.js [hunk 2]
```diff
@@ -6,12 +6,15 @@
...
-import { Tabs, Tab, Divider, colors } from '@material-ui/core';
+import { Tabs, Tab, Divider, colors, Paper, Input } from '@material-ui/core';
 import ApprovalCard from '../components/requests/approvalsCard';
 import Pagination from '../components/common/Pagination';
 import Header from '../components/common/SettingsHeader';
 import { getManagerRequests } from '../redux/actions/requestApprovalAction';
+import searchAction from '../redux/actions/searchAction';
 
 const drawerWidth = 240;
 export default function ApprovalsTable({ history }) {
```
Comment (bdushimi) [inline src/views/approvalsView.js:21]: @erickyvand any reason why this PR contains changes in the unrelated files i.e. approvalsView?

File: src/views/approvalsView.js [hunk 3]
```diff
@@ -82,9 +103,53 @@ export default function ApprovalsTable({ history }) {
...
+
+  const indexOfLastSearchPage = currentSearchPage * tripsSearchPerPage;
+  const indexOfFirstSearchPage = indexOfLastSearchPage - tripsSearchPerPage;
+  const currentSearchTrips = allResults.slice(indexOfFirstSearchPage, indexOfLastSearchPage);
+
+  const paginateSearch = PageSearchNumber => {
+    setCurrentSearchPage(PageSearchNumber);
+  };
+
+	const handleChange = e => {
```
Comment (bdushimi) [inline src/views/approvalsView.js:127]: @erickyvand which change is this handleChange function handling? applicable elsewhere.

File: src/views/approvalsView.js [hunk 4]
```diff
@@ -82,9 +103,53 @@ export default function ApprovalsTable({ history }) {
...
+		setLocation(e.target.value);
+		setName(e.target.value);
+		setTheStatus(e.target.value);
+		setTime(e.target.value);
+	};
+
+	const handleclick = () => {
```
Comment (bdushimi) [inline src/views/approvalsView.js:134]: @erickyvand may you specify on which component does handleClick function handle? Applicable elsewhere

**Review by bdushimi — CHANGES_REQUESTED** [review]

File: src/views/approvalsView.js (hunk 2 above)
Comment (erickyvand) [reply src/views/approvalsView.js:21]: @bdushimi, It contains changes because that file contains requests and I needed to search in those requests

File: src/views/approvalsView.js (hunk 3 above)
Comment (erickyvand) [reply src/views/approvalsView.js:127]: This function is handling input changes in the search input field

File: src/views/approvalsView.js (hunk 4 above)
Comment (erickyvand) [reply src/views/approvalsView.js:134]: The search results are being displayed in approvalsView component, when no search has been made it will show requests but when a search has been made it will show the search results

**Review by bdushimi — APPROVED** [review]
